src/Popover/Popover.test.tsx

```
@@ -0,0 +1,164 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+import { Popover } from './Popover';
+import 'jest-styled-components';
+
+describe('Popover Component', () => {
+    const triggerText = 'Open Popover';
+    const contentText = 'This is the popover content.';
+    const PopoverUncontrolled = (
+        <Popover
+            trigger={<button>{triggerText}</button>}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)